starter-code/apple-pie-starter-code/main.css

```diff
@@ -0,0 +1,62 @@
+body {
```

Comment:
I would not put so many styles on the body.  If you do this in a bigger project, it will apply these changes to the whole project.  You probably did this to center the image, but it can be done in other ways.